src/openrct2-ui/CMakeLists.txt

```diff
@@ -9,10 +9,10 @@ option(DISABLE_OPENGL "Disable OpenGL support.")
 
 # Third party libraries
 if (MSVC)
-    find_package(jansson REQUIRED)
```

Comment:
Why remove the dependency for the MSVC target?

Reply:
Sorry, I moved from a FindFile.cmake to a PKG way. Will update this.